Platforms/Export-Import-Platform.ps1

```diff
@@ -74,6 +78,32 @@ Function Test-CommandExists
 
 If (Test-CommandExists Invoke-RestMethod)
 {
+If($DisableSSLVerify)
+{
+	try{
+		Write-Warning "It is not Recommended to disable SSL verification" -WarningAction Inquire
+		# Using Proxy Default credentials if the Server needs Proxy credentials
+		[System.Net.WebRequest]::DefaultWebProxy.Credentials = [System.Net.CredentialCache]::DefaultCredentials
+		# Using TLS 1.2 as security protocol verification
+		[System.Net.ServicePointManager]::SecurityProtocol = [System.Net.SecurityProtocolType]::Tls12 -bor [System.Net.SecurityProtocolType]::Tls11
+		# Disable SSL Verification
+		[System.Net.ServicePointManager]::ServerCertificateValidationCallback = { $DisableSSLVerify }
+	} catch {
+		Write-Error "Could not change SSL validation"
+		Write-Error (Join-ExceptionMessage $_.Exception) -ErrorAction "SilentlyContinue"
```

Comment:
Same comment here about the join-ExceptionMessage

Reply:
Added in function Join-ExceptionMessage